src/pages/[platform]/start/quickstart/vite-react-app/index.mdx

```diff
@@ -28,235 +30,307 @@ export function getStaticProps(context) {
 
-```bash title="Terminal" showLineNumbers={false}
-npm create vite@latest react-amplify-gen2 -- --template react-ts
-```
-
-This creates a new React app in a directory called react-amplify-gen2. You can then run the following commands to switch into the new `react-amplify-gen2` directory and install the project dependencies:
-
-```bash title="Terminal" showLineNumbers={false}
-cd react-amplify-gen2
-npm install
-```
+Next, go to the AWS Amplify console to deploy the application.
 
-## Create a backend
+<ExternalLinkButton
+  size="large"
+  href='https://us-east-1.console.aws.amazon.com/amplify/create/repo-branch'
+>
+<IconAmplify />
+Deploy starter template
+</ExternalLinkButton>
 
-The easiest way to get started with AWS Amplify is through npm with `create-amplify`.
+Select **Start with an existing app** > **GitHub**, pick the starter repository, and hit "Deploy".
 
-```bash title="Terminal" showLineNumbers={false}
-npm create amplify@beta
-```
+<video autoPlay={true} muted={true} loop={true} width="100%" playsInline={true}>
+  <source src="/images/gen2/getting-started/react/deploy.mp4" />
+</video>
 
-```console title="Terminal" showLineNumbers={false}
-? Where should we create your project? (.) # press enter
-```
-
-Running this command will scaffold a lightweight Amplify project in your current project with the following files:
+While you wait for the build (~5 min), let's take a tour on the key files included in the starter repository:
 
 ```text
-├── amplify/
-│   ├── auth/
+├── amplify/ # Folder containing your Amplify backend configuration
+│   ├── auth/ # Definition for your auth backend
 │   │   └── resource.ts
-│   ├── data/
+│   ├── data/ # Definition for your data backend
 │   │   └── resource.ts
-│   ├── backend.ts
-│   └── package.json
-├── node_modules/
-├── .gitignore
-├── package-lock.json
+│   └── backend.ts
+├── src/ # React UI code
+│   ├── App.tsx # UI code to sync todos in real-time
+│   ├── index.css # Styling for your app
+│   └── main.tsx # Entrypoint where the Amplify client library is configured
 ├── package.json
 └── tsconfig.json
 ```
 
-## Start local dev server
+Once the build has completed view the newly deployed app by selecting "View production URL".
+
+<video autoPlay={true} muted={true} loop={true} width="100%" playsInline={true}>
+  <source src="/images/gen2/getting-started/react/hosted-app.mp4" />
+</video>
+
+## Build and connect to a database
 
-Let's start a local dev server for your app development. For the frontend, run `npm run dev` to spin up a `localhost` dev server.
+The starter application already has pre-written code to give you a real-time feed of all todos and the ability to add new todos. Let's learn how to enhance the functionality with delete operations.
+
+First, clone your repository locally by following instructions on your GitHub repo page:
 
 ```bash title="Terminal" showLineNumbers={false}
-npm run dev
+git clone https://github.com/<GITHUB_USER>/gen2-starter.git
```

Comment:
```suggestion
git clone https://github.com/<github-user>/gen2-starter.git
```

this would be snazzy with `gh` 

```bash
gh repo create --template renebrandel/gen2-starter gen2-starter --public
```